### PR TITLE
Update metadata for Vox Pupuli release

### PR DESCRIPTION
**Pull request by pillarsdotnet**
Update urls and version numbers in the `metadata.json` file.

### CODE REVIEW

**Comment by pillarsdotnet** [comment]
I don't think the test-failures are related to the metadata change.

File: metadata.json [hunk 1]
```diff
@@ -59,7 +60,7 @@
     },
     {
       "name": "puppetlabs/java",
-      "version_requirement": ">= 1.0.1 < 4.0.0"
+      "version_requirement": ">= 1.0.1 < 5.0.0"
```
Comment (Dan33l) [inline metadata.json]: not sure failing tests are related to this. But to be sure, can you rollback to `< 4.0.0`  ?

Comment (pillarsdotnet) [reply metadata.json]: Done.

**Comment by pillarsdotnet** [comment]
@Dan33l Tests passed.

File: metadata.json (hunk 1 above)
Comment (pillarsdotnet) [reply metadata.json]: Still fails, because java 4 requires stdlib 5.  Looks like we need an update of java?

**Pull request merged**